src/components/common/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './LargeButton';
+export * from './NavbarButton';
+export * from './SmallButton';
```

Comment:
Created an `index.js` file for all common components. Now to import any of them we can just do 
`import {ComponentName} from '../components/common'` instead of having to drill into each specific file.